cmd/osm-controller/osm-controller.go

```diff
@@ -333,3 +337,44 @@ func getOSMControllerPod(kubeClient kubernetes.Interface) (*corev1.Pod, error) {
 
 	return pod, nil
 }
+
+// StartGlobalLogLevelHandler registers a listener to meshconfig events and log level changes,
+// and applies new log level at global scope
+func StartGlobalLogLevelHandler(cfg configurator.Configurator, stop <-chan struct{}) {
```

Comment:
Move this to its own file `logging.go` for log management.

Reply:
thought of that. will do.